### PR TITLE
Mediator as a net standard library instead of net core app

### PR DESCRIPTION
**Pull request by bcornett**

### CODE REVIEW

File: Turner.Infrastructure.Mediator.Tests/Fakes/FakeDbContext.cs [hunk 1]
```diff
@@ -7,14 +8,21 @@ namespace Turner.Infrastructure.Mediator.Tests.Fakes
...
+        }
+
         protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
             optionsBuilder.ConfigureWarnings(builder => builder.Ignore(InMemoryEventId.TransactionIgnoredWarning));
             optionsBuilder.UseInMemoryDatabase();
             base.OnConfiguring(optionsBuilder);
         }
 
-        public override DatabaseFacade Database => new FakeDatabaseFacade(this);
```
Comment (bcornett) [inline Turner.Infrastructure.Mediator.Tests/Fakes/FakeDbContext.cs:17]: **Annotation**
Was newing up the facade every time this was accessed.

File: Turner.Infrastructure.Mediator/Turner.Infrastructure.Mediator.csproj [hunk 2]
```diff
@@ -1,18 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (bcornett) [inline Turner.Infrastructure.Mediator/Turner.Infrastructure.Mediator.csproj:4]: **Annotation**
Changed from core app to standard library

File: Turner.Infrastructure.Mediator.Tests/Fakes/FakeDbContext.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Diagnostics;
```
Comment (MatthewPuneky) [inline Turner.Infrastructure.Mediator.Tests/Fakes/FakeDbContext.cs:2]: Is this using required?

Comment (bcornett) [reply Turner.Infrastructure.Mediator.Tests/Fakes/FakeDbContext.cs:2]: Yes, upgraded the package and the new one changes where `InMemoryEventId` is housed.
image

Comment (MatthewPuneky) [reply Turner.Infrastructure.Mediator.Tests/Fakes/FakeDbContext.cs:2]: Ah okay

**Review by MatthewPuneky — APPROVED** [review]

**Pull request merged**